utils.ts

```diff
@@ -128,14 +128,16 @@ export const fetchPelias = async (
  */
 export const arePointsRoughlyEqual = (a: Position, b: Position): boolean => {
   // 4 decimal places is approximately 10 meters, which is acceptable error
-  const aRounded = a.map((point: number): number =>
-    parseFloat(point.toFixed(4))
+  const aRounded = a?.map((point: number): number =>
```

Comment:
What problem are we solving with the optional chaining? It shouldn't be necessary if the TypeScript types on the arguments are correct. I'm guessing that we're getting an API response back somewhere that doesn't always conform to the shape we've defined with TypeScript. Perhaps we should update that type so TypeScript can correctly show us where we're using the API response incorrectly. The answer in this function might be just to make the arguments nullable.